setup.py

```diff
@@ -86,7 +86,7 @@ def _core_version(plugin_version: str = _plugin_version()) -> str:
         f"dbt-postgres~={_core_version()}",
         # dbt-redshift depends deeply on this package. it does not follow SemVer, therefore there have been breaking changes in previous patch releases
         # Pin to the patch or minor version, and bump in each new minor version of dbt-redshift.
-        "redshift-connector==2.0.915",
+        "redshift-connector<=2.0.918, >=2.0.913",
```

Comment:
Didn't we have an issue with 2.0.914? I seem to remember jumping from 2.0.913 to 2.0.915.

Reply:
good memory! https://github.com/dbt-labs/dbt-redshift/pull/601#issuecomment-1718442908